### PR TITLE
ENH added zip-cos6-x86_64

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by isuruf** [comment]
Why do we need this when there's `zip-feedstock`?

**Comment by beckermr** [comment]
I’m copying everything from defaults. I didn’t try to figure out if we needed them or not.

File: recipes/zip-cos6-x86_64/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  summary: (CDT) A file compression and packaging utility compatible with PKZIP
+  description: |
+    The zip program is a compression and file packaging utility.  Zip is analogous
+    to a combination of the UNIX tar and compress commands and is compatible with
+    PKZIP (a compression and file packaging utility for MS-DOS systems).  Install
+    the zip package if you need to compress files using the zip program.
+
+extra:
+  recipe-maintainers:
+    - conda-forge/Core
```
Comment (isuruf) [inline recipes/zip-cos6-x86_64/meta.yaml:38]: Please add yourself and anybody else interested.

Comment (beckermr) [reply recipes/zip-cos6-x86_64/meta.yaml:38]: Why not core?

Comment (beckermr) [reply recipes/zip-cos6-x86_64/meta.yaml:38]: Nobody is interested but this has to be done. Thus I think this falls to core

Comment (isuruf) [reply recipes/zip-cos6-x86_64/meta.yaml:38]: I don't want to be a maintainer of this useless feedstock. So, please leave me out of it.

Comment (beckermr) [reply recipes/zip-cos6-x86_64/meta.yaml:38]: Got it. Please send a list of cdts you think are useful so that we can avoid this.

**Pull request merged**